.github/workflows/ci.yml

```diff
@@ -434,6 +434,57 @@ jobs:
               ./build_tools/scripts/check_vulkan.sh
```

Comment:
[Putting comment here for threading]

> I double-checked with Dan and these tests run over a couple of hours (more models have been added recently).
>
> @GMNGeoffrey @pzread How do we want to handle test failures if we run this as post-submit?

I'm not quite sure what you mean by "handle". Could you elaborate?

If these are taking hours then we should run them on a schedule rather than on every commit. It sounds like until recently this could run in single-digit minutes? Can we extract the set of models that do run quickly and run those more often?

Are these tests making individual use of the network? That will give us flakiness that scales with the number of tests.

Reply:
> I'm not quite sure what you mean by "handle". Could you elaborate?

The goal is to catch regressions so if we see failures, how can we be notified of them, retrieve the results, etc. I'm wondering if there's a standard way of doing this using the current infra.

> If these are taking hours then we should run them on a schedule rather than on every commit. It sounds like until recently this could run in single-digit minutes? Can we extract the set of models that do run quickly and run those more often?

We'll create a separate set of models that can run within minutes for presubmit.

> Are these tests making individual use of the network? That will give us flakiness that scales with the number of tests.

@dan-garvey can comment but it looks like once it's downloaded all of the artifacts, it benchmarks them locally.

Reply:
Yeah to elaborate on anush's point, it downloads artifacts based on the pinned shark sha, but then compiles them using iree tools, and then runs them. (This particular ci does not benchmark them but could be changed to)

Reply:
If it's running on post-submit, let's add the benchmark flag. @GMNGeoffrey @pzread Is there a specific bucket we should upload the results and build artifacts to?

Reply:
@GMNGeoffrey @pzread friendly bump

Reply:
After internal discussions, let's get this submitted and running regularly to monitor stability, then add features incrementally.

Reply:
Great, can someone with permission trigger the ci.yml workflow for this patch so I can see if it runs successfully?

Reply:
Can you please sync this with mainline and fix the runner specification to match the other jobs (in particular, targeting only the prod environment). I will add you as a collaborator so workflows run automatically from now on